bin/filter_header.pl

```diff
@@ -0,0 +1,221 @@
+use strict;
+use warnings;
+use autodie;
+use Getopt::Long;
+use Carp;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use st::api::lims;
+
+our $VERSION = '0';
+
+
+my ($rpt,$run,$pos,$tag,$af1,$af2,$lims,$sample,$library,$study,);
+
+&GetOptions("rpt:s"  => \$rpt,);
+
+# Parse file name
+if($rpt =~ /^(\d+)_(\d)(_(\w+))?(\#(\d+))?(_(\w+))?$/mxs){
```

Comment:
Should we use existing code to get this rpt parsing? So we only have one place to fix/update it....